### PR TITLE
Create ExcelNPOIStorage

### PR DESCRIPTION
**Pull request by JasonCSharpCoder**
I change your code in ExcelNPOIStorage.cs file like upon, because by your code I get a bug about values to Mapper object, because when my cell in excel is empty the cellValues will not contain it, the length of values will shorter comparing with mapping RecordFieldCount. exception is out of range

### CODE REVIEW

**Comment by Tawhid1907** [comment]
Please include this pull request as we are also facing the same problem with empty cells in excel

**Comment by mcavigelli** [comment]
Thank you for using FileHelpers.

I don't understand the problem in full detail:

1. We would like to be able to reproduce your problem: please include a unittest demonstrating the problem or include the file you have problems to read.

2. Line 13 of your method contains a comment "TO:DO": could please elaborate what has to be done?

File: ExcelNPOIStorage [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                return null;
+
+            if (numberOfCols == 1) {
+                IRow row = mSheet.GetRow(rowNum);
+
+                ICell cell = HSSFCellUtil.GetCell(row, startCol);
+                return new object[] {NPOIUtils.GetCellValue(cell)};
+            }
+            else {
+                //TO:DO
```
Comment (mcavigelli) [inline ExcelNPOIStorage]: Could you please elaborate what has to be done?

File: ExcelNPOIStorage [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                    if (NPOIUtils.GetCellValue(cell) == null) {
+                        cell.SetCellValue (" ");
+                    }
+                }
+
+                CellRangeAddress range = new CellRangeAddress(rowNum, rowNum, startCol, startCol + numberOfCols - 1);
+
+                CellWalk cw = new CellWalk(mSheet, range);
+
+                //cw.IsTraverseEmptyCells();
```
Comment (mcavigelli) [inline ExcelNPOIStorage]: Is there any reason why this method is contained as a comment? Or could it be removed?

Comment (JasonCSharpCoder) [inline ExcelNPOIStorage]: yes it could be removed ,sorry to commit this code like this , i will notify it next time.

Comment (mcavigelli) [inline ExcelNPOIStorage]: Thank you for your explanations, I got closer to understand the problem.
Attaching a file that reproduces the problem would be very helpful.
A code snippet with how you are calling the library would be helpful too.

Thank you for supporting us, Matthias

Comment (JasonCSharpCoder) [inline ExcelNPOIStorage]: this is also my pleasure ,hope you could make it better!

**Comment by mcavigelli** [comment]
Seems to be solved with #219, which is included in master.

**Pull request closed without merging**